.travis.yml

```diff
@@ -1,46 +1,57 @@
 sudo: false
 language: python
-python:
- - "2.7"
- - "3.3"
- - "3.4"
- - "3.5"
- - "3.6"
+
+python: "3.6"
+
 env:
   matrix:
-   - DJANGO="Django<1.9"
```

Comment:
Me too :) But that sounds unnecessary; it's a lot of extra time/effort to spend on the one and only 3.3 test which will no longer be supported in just 4 months. Requiring 3.4 is very reasonable for allauth to do.